smack-core/src/main/java/org/jivesoftware/smack/XMPPConnection.java

```diff
@@ -45,7 +45,6 @@
  * // Start a new conversation with John Doe and send him a message.
  * ChatManager chatManager = ChatManager.getInstanceFor(con);
  * chatManager.addIncomingListener(new IncomingChatMessageListener() {
- *     @Override
```

Comment:
Please do not mix documentation changes with code changes. Rather create a new PR for changes unrelated to documentation.

Reply:
Anything touching a *.java (source code) file is considered a code change?